### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by Jordan-M-Schultz**

### CODE REVIEW

File: RedcapNotifications.php [hunk 1]
```diff
@@ -90,8 +90,8 @@ function redcap_every_page_top($project_id) {
             'UserRights/index.php'
         ];
 
-//        if(in_array(PAGE, $allowed_pages))
-//            $this->injectREDCapNotifs();
+        if(in_array(PAGE, $allowed_pages))
```
Comment (ihabzee) [inline RedcapNotifications.php:93]: why is `$allowed_pages` not an EM system setting?

**Review by ihabzee — COMMENTED** [review]
I dismissed a new notification. Now every time I refresh any redcap page a new record will be created in dismiss redcap project for the notification I dismissed.

File: assets/scripts/RCNotif_old.js [hunk 2]
```diff
@@ -0,0 +1,263 @@
+// //NOTIF TEMPLATES
```
Comment (ihabzee) [inline assets/scripts/RCNotif_old.js:1]: why keep this file?

Comment (Jordan-M-Schultz) [reply assets/scripts/RCNotif_old.js:1]: I suppose I can remove

**Pull request merged**